docs/madr/decisions/030-extended-global-insights.md

```diff
@@ -78,10 +83,18 @@ should look like this:
       }
    },
    "dataplanes": {
```

Comment:
**Question**: As far as I know, exactly this sub structure appears on `/mesh-insights` but it’s on the `dataplanesByType` field. Should we use this field name here, too?

Reply:
I don't have any strong opinion on this. We probably should be consistent with `mesh-insight`

Reply:
updated

Reply:
Same, I’m fine with either. Just figured I bring it up.

Reply:
but, when I think more about this, using `dataplanesByType` would be not consistent with the rest of the `GlobalInsight` response

Reply:
so in the end I would go with `dataplanes`

Reply:
Fine by me, too.